src/deadline/unreal_submitter/unreal_logger/logger.py

```diff
@@ -0,0 +1,33 @@
+import os
+import json
+import logging
+import logging.config
+from typing import Optional
+
+from deadline.unreal_submitter.unreal_logger.handlers import UnrealLogHandler
+
+
+with open(f"{os.path.dirname(__file__)}/config.json", "r") as f:
+    LOGGER_CONFIG = json.load(f)
+
+
+def get_logger() -> logging.Logger:
+
+    logging.config.dictConfig(LOGGER_CONFIG)
+
+    unreal_logger = logging.getLogger("unreal_logger")
```

Comment:
We never actually add our UnrealLogHandler with [addHandler](https://docs.python.org/3/library/logging.html#logging.Logger.addHandler) it looks like?  Again maybe there's a good reason, but it seems like we're doing more work to manage the handler than I'd expected we'd need to..

Reply:
we setup our custom "unreal_logger" logger with config.json file that lives inside same directory and set up it with `logging.config.dictConfig`. Here we describe the formatters, handlers and the logger itself. I think its better to keep configuration in the appropriate file instead of configuring in the code. What do you think?

Reply:
I was more surprised we didn't add our handler to the logger with addHandler, then just go through the normal python logger.info type methods.  Commented above about this, but I think this is also why we're ending up doing the "add 2 log handlers, remove 1" steps currently, so yeah it might look cleaner if we did it as code and only added one, or had two data files potentially.